### ISSUE TITLE
The Performance on Cityscapes?

### ISSUE THREAD

**Issue by FengLoveBella** [milongo/CASENet#1]
@milongo What's the performance of your CASENet code on Cityscapes or SBD? Can you share the experimental result or the train model? Tks very much!

**Comment by milongo** [comment]
Hi @zhoufengbuaa, the truth is that once I got the model working I couldn't find the time to test it on Cityscapes. As it is, the download link includes the SBD dataset that the code can train with (with a lot of other irrelevant files and folders, I'm afraid I would have to clean those if I have the time), but I haven't gotten around to training it and writing a test script...

**Comment by FengLoveBella** [comment]
@milongo So you cannot assume that your code is ok or not?

**Comment by milongo** [comment]
@zhoufengbuaa the code is an implementation of the CASENet network and loss function with sliced concatenation instead of shared concatenation. I believe the code is correct in that sense, but I haven't tested it so I can't say if it achieves the performance reported on the paper.

**Comment by milongo** [comment]
@zhoufengbuaa I have updated the README to clarify this.

**Comment by FengLoveBella** [comment]
@milongo ok, thank you very much. I appreciate your confidence （smile，smile，smile）

**Comment by FengLoveBella** [comment]
@milongo Another quetion, what's the batch size of this code? How many GPUs are needed for this program？

**Comment by FengLoveBella** [comment]
@milongo The dataset you proposed in your code, I think it is not OK. All the image in the folders are none. All pixels in these images are zero. Could you check it? Thank you very much.

**Comment by milongo** [comment]
@zhoufengbuaa As far as I remember, it's not that they're empty. Pixels are labeled as either 0 or 1 for each class, but white in images is a value of 255, so they may appear black. Can you check if what I say is right?

**Comment by FengLoveBella** [comment]
@milongo I am some sure.  Some picture in your dataset is none, I use matlab to open some pixel, and I found the pixels are all zero. Could you check it, thank you very much.

**Comment by FengLoveBella** [comment]
@milongo By the way, could you share your_contours.txt and weights/casenet_weights.pth in your code? Thank you very much!

**Comment by milongo** [comment]
@zhoufengbuaa What does "none" mean? Which images were "none"? If you opened some images only, it's possible that you got all zero. Not all SBD images have contours for all classes. 

I have lost access to the machine I developed this in, and as a result I don't have the text file or weights. The weights were just random weights anyway. 

I suggest you ignore my SBD dataset, download the original dataset, and create a new DataLoader.

**Comment by FengLoveBella** [comment]
@milongo Ok, Thank you very much for your reply. Looking forward to your to-do experiments. Smile

**Comment by FengLoveBella** [comment]
@milongo Hi, I found there is some difference between your code and the released caffe code, such as the loss implementation, the output of the channels number.  And I run your code, I found the loss is very big and seem not to decrease during training. Do you have any suggestion about training your code?
